pkgs/tools/audio/bpm-tools/default.nix

```diff
@@ -12,14 +22,22 @@ stdenv.mkDerivation rec {
     sha256 = "151vfbs8h3cibs7kbdps5pqrsxhpjv16y2iyfqbxzsclylgfivrp";
   };
 
+  nativeBuildInputs = [ makeWrapper ];
+
   patchPhase = ''
     patchShebangs bpm-tag
     patchShebangs bpm-graph
   '';
```

Comment:
This should be removed. All installed executables will have shebangs patched automatically, and this also breaks `patches` attribute.